driver-core/src/main/java/com/datastax/driver/core/TypeCodec.java

```diff
@@ -1079,6 +1093,11 @@ private static Long parseTime(String s) throws IllegalArgumentException {
 
         @Override
         public Long parse(String value) {
+            // enclosing single quotes required, even for long literals
```

Comment:
Yes. I noticed with cqlsh that time literals in the long form must be enclosed in single quotes, while for date and timestamp literals in the long form, this is optional. E.g:
```sql
cqlsh:test> create table t1 (pk int primary key, c1 date, c2 time, c3 timestamp);
cqlsh:test> insert into t1 (pk, c1, c2, c3) values (0, 0, 0, 0);
InvalidRequest: code=2200 [Invalid query] message="Invalid INTEGER constant (0) for "c2" of type time"
cqlsh:test> insert into t1 (pk, c1, c2, c3) values (0, 0, '0', 0);
cqlsh:test> insert into t1 (pk, c1, c2, c3) values (1, '0', '0', '0');
cqlsh:test> select * from t1;

 pk | c1          | c2                 | c3
----+-------------+--------------------+--------------------------
  1 | -2147483648 | 00:00:00.000000000 | 1970-01-01 01:00:00+0100
  0 | -2147483648 | 00:00:00.000000000 | 1970-01-01 01:00:00+0100

(2 rows)
cqlsh:test>
```